modules/local/bowtie2_assembly_align.nf

```diff
@@ -1,6 +1,6 @@
 // Influenced by https://github.dev/nf-core/mag but modified ouput handling
 process BOWTIE2_ASSEMBLY_ALIGN {
-    tag "${assembly_meta.id}-vs-${reads_meta.id}"
+    tag "${meta.id}-vs-${meta.id}"
```

Comment:
i guess this is about your reciprocal comment in the PR description? if so you can ignore

Reply:
but then maybe the question is how will this work eventually? will you have to pass two different metas?

Reply:
yeah they will be the same but eventually with reciprocal mapping it will have to be setup in a way where they aren't the same sample, so yes I would have to pass different metas. I think I'll have to think further about how to implement that for passing in the different metas

Reply:
this could be controlled in the future possibly with parameterization but I'm a bit of a ways from getting to the reciprocal mapping for time-series